utils.c

```diff
@@ -71,23 +72,25 @@ gboolean janus_strcmp_const_time(const void *str1, const void *str2) {
 }
 
 guint32 janus_random_uint32(void) {
-	return g_random_int();
+	guint32 ret = 0;
+	if (RAND_bytes((void *)&ret, sizeof(ret)) != 1) {
+		JANUS_LOG(LOG_FATAL, "\tOpenSSL RAND_bytes() failed\n");
+		exit(1);
```

Comment:
I don't like the `exit(1)` here at all... This will cause the program to just exit while in use, which should never happen. I'd much rather fallback to the unsafe glib random-int implementation when `RAND_bytes` fails (maybe with a warning printed out rather than a fatal error).

Reply:
ok